src/components/Badge/Badge.module.scss

```diff
@@ -8,9 +6,9 @@ $embeddedPosition: 15%;
   .badge {
     z-index: 1;
     position: absolute;
-    border: 2px solid transparent;
+    border: 2px solid;
```

Comment:
it's look ok without the transparent? no black border or something?

Reply:
It was overriden by `border-color` which is standing after it
<img width="723" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/423035f1-a03c-4ff8-9d1f-6fabedb8b8e4">